Add a signup form for the mailchimp announce list

plexus opened the pull request:
We now have a mailing list so people who like to hear about future
workshops can easily subscribe. This adds a small form at the bottom to
subscribe to this list.

plexus committed:
Add a signup form for the mailchimp announce list

We now have a mailing list so people who like to hear about future
workshops can easily subscribe. This adds a small form at the bottom to
subscribe to this list.

Malwine commented:
I love the idea. Though I would change the position of the form ad put it to the top.
I will branch off for the idea.

Malwine committed:
Update text and move the form to the top of the page

Malwine committed:
Update attention box css for sign up for newsletter and workshop

Malwine committed:
Clean up MailChimp template
- take out link to MailChimp css
- use bootstrap css for form and button
- delete css comments

Malwine committed:
Change text for mailing list box

Malwine committed:
Merge pull request #15 from clojurebridge-berlin/update_and_change_position_for_newsletter_signup

Update and change position for newsletter signup
